tests/qunit/js/editor-base.spec.js

```diff
@@ -1,67 +1,53 @@
-/* global ElementorConfig */
-import editorBase from './../../../assets/dev/js/editor/editor-base';
+import EditorBase from './../../../assets/dev/js/editor/editor-base';
 
-const App = editorBase.extend();
+export default class EditorBaseTest extends EditorBase {
```

Comment:
Rename also the file name.
```suggestion
export default class EditorTest extends EditorBase {
```